.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Bug report
+labels: bug
+description: 'Description of a bug report'
+
+body:
+- type: dropdown
+  id: occurence
+  attributes:
+    label: 'How often bug is revealed:'
+    options: ['- Always', '- Often', '- Rare']
+  validations:
+    required: true
+
+- type: checkboxes
+  id: modules
+  attributes:
+    label: Which part is affected?
+    description: Select all parts affected by this bug.
+    options:
+      - label: tests
+      - label: infrastructure
+      - label: documentation
+      - label: allocator
+
```

Comment:
Can we add more options to specify library subcomponents? Or at least add option "other".

Reply:
Agree with @omalyshe. The rest looks good, thank you!

Reply:
Sure, we can add more subcomponents, but I don't have much knowledge about oneTBB.  I've added the "other" option, but we can extend it in the future.